app/assets/stylesheets/custom.scss

```diff
@@ -0,0 +1,45 @@
+.navbar {
+  margin-top: 0;
+  margin-bottom: 0;
+}
+
+body {
+  background: image-url("raw/goofy-dad-bg.jpg") no-repeat center center fixed;
+  -webkit-background-size: cover;
```

Comment:
Properties should be ordered background, -moz-background-size, -o-background-size, -webkit-background-size, background-size